### ISSUE TITLE
Animation doesn't work for inventory sprites

### ISSUE THREAD

**Issue by Macoron** [space-wizards/space-station-14#5104]
## Description

When you take an animated item in hands or place it in inventory animation stops playing. Sprites placed back in world works alright. Check matches or cigarettes for example.

**Screenshots**

https://user-images.githubusercontent.com/6161335/139538131-6791a9ea-7807-4582-b79e-fbf1381140a1.mp4

**Comment by Macoron** [comment]
Looks like this is robust engine problem. Probably SpriteTree culls SpriteComponent placed inside containers. Check [SpriteSystem ](https://github.com/space-wizards/RobustToolbox/blob/8b462cc099ab265e6628a008428c04b8ce1405e4/Robust.Client/GameObjects/EntitySystems/SpriteSystem.cs#L53)for more info.

Replacing `SpriteTree` with a simple query fix this bug:
```
        public override void FrameUpdate(float frameTime)
        {

            foreach (var comp in EntityManager.EntityQuery<SpriteComponent>())
            {
                comp.FrameUpdate(frameTime);
            }
        }
```

**Comment by mirrorcult** [comment]
fixed by space-wizards/RobustToolbox#2181
